packages/core/integration-tests/test/output-formats.js

```diff
@@ -1151,9 +1151,10 @@ describe('output formats', function () {
 
       for (let bundle of [async1Bundle, async2Bundle]) {
         // async import both bundles in parallel for performance
+        // esm load function public id = "$fPYKz"
         assert(
           new RegExp(
-            `import\\("\\./" \\+ .+\\.resolve\\("${sharedBundle.publicId}"\\)\\),\\n\\s*import\\("./" \\+ .+\\.resolve\\("${bundle.publicId}"\\)\\)`,
+            `\\$fPYKz\\("${sharedBundle.publicId}"\\),\\n\\s*\\$fPYKz\\("${bundle.publicId}"\\)`,
```

Comment:
This is passing locally which makes me think that the asset has a different publicId in CI. Is this expected?